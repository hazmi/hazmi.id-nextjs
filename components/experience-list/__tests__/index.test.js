import React from 'react'
import renderer from 'react-test-renderer'
import ExperienceList from '../index'

describe('With Snapshot Testing', () => {
  it('show the correct content', () => {
    const component = renderer.create(<ExperienceList />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})