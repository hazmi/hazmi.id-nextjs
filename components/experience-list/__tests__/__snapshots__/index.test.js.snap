// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing show the correct content 1`] = `
<div
  className="experience-list"
>
  <section
    className="experience-list__item"
  >
    <img
      alt="Logo Mango33"
      src="/static/work/mango33.png"
    />
    <div
      className="experience-list__item-content"
    >
      <h5
        className="experience-list__item-date"
      >
        May 2018 — Present
      </h5>
      <h3
        className="experience-list__item-title"
      >
        Tech Lead / IT Director
      </h3>
      <h4
        className="experience-list__item-company"
      >
        <span>
          at
        </span>
         Mango33 Pte Ltd
      </h4>
      <div
        className="experience-list__item-info"
      >
        <p>
          Working on HR mobile application.
        </p>
      </div>
    </div>
  </section>
  <section
    className="experience-list__item"
  >
    <img
      alt="Logo CelliHealth"
      src="/static/work/cellihealth.png"
    />
    <div
      className="experience-list__item-content"
    >
      <h5
        className="experience-list__item-date"
      >
        May 2016 — May 2018 · 2yrs 1mo
      </h5>
      <h3
        className="experience-list__item-title"
      >
        Tech Lead / IT Director
      </h3>
      <h4
        className="experience-list__item-company"
      >
        <span>
          at
        </span>
         CelliHealth Pte Ltd
      </h4>
      <div
        className="experience-list__item-info"
      >
        <p>
          Working on Healthcare mobile application.
        </p>
      </div>
    </div>
  </section>
  <section
    className="experience-list__item"
  >
    <img
      alt="Logo KMK"
      src="/static/work/kmk.png"
    />
    <div
      className="experience-list__item-content"
    >
      <h5
        className="experience-list__item-date"
      >
        May 2015 — May 2016 · 1yr 1mo
      </h5>
      <h3
        className="experience-list__item-title"
      >
        Lead Front-end Engineer
      </h3>
      <h4
        className="experience-list__item-company"
      >
        <span>
          at
        </span>
         PT. Kreatif Media Karya (KMK)
      </h4>
      <div
        className="experience-list__item-info"
      >
        <p>
          Working on Liputan6 and Vidio desktop and mobile website.
        </p>
      </div>
    </div>
  </section>
  <section
    className="experience-list__item"
  >
    <img
      alt="Logo Graipe Labs"
      src="/static/work/graipe.png"
    />
    <div
      className="experience-list__item-content"
    >
      <h5
        className="experience-list__item-date"
      >
        Jan 2013 — Apr 2015 · 2yr 3mos
      </h5>
      <h3
        className="experience-list__item-title"
      >
        Tech Lead / VP Technology
      </h3>
      <h4
        className="experience-list__item-company"
      >
        <span>
          at
        </span>
         Graipe Labs
      </h4>
      <div
        className="experience-list__item-info"
      >
        <p>
          Working on OTA websites and mobile applications.
        </p>
      </div>
    </div>
  </section>
  <section
    className="experience-list__item"
  >
    <img
      alt="Logo Kitook"
      src="/static/work/kitook.png"
    />
    <div
      className="experience-list__item-content"
    >
      <h5
        className="experience-list__item-date"
      >
        Aug 2012 — Apr 2014 · 1yr 9mos
      </h5>
      <h3
        className="experience-list__item-title"
      >
        CTO
      </h3>
      <h4
        className="experience-list__item-company"
      >
        <span>
          at
        </span>
         PT. VG Wisata Indonesia (Kitook)
      </h4>
      <div
        className="experience-list__item-info"
      >
        <p>
          Working on OTA website.
        </p>
      </div>
    </div>
  </section>
  <section
    className="experience-list__item"
  >
    <img
      alt="Logo Gonla"
      src="/static/work/gonla.png"
    />
    <div
      className="experience-list__item-content"
    >
      <h5
        className="experience-list__item-date"
      >
        Oct 2011 — Aug 2012 · 11mos
      </h5>
      <h3
        className="experience-list__item-title"
      >
        Senior Web Developer
      </h3>
      <h4
        className="experience-list__item-company"
      >
        <span>
          at
        </span>
         PT. Perjalanan Wisata (Gonla)
      </h4>
      <div
        className="experience-list__item-info"
      >
        <p>
          Working on OTA website.
        </p>
      </div>
    </div>
  </section>
  <section
    className="experience-list__item"
  >
    <img
      alt="Logo XHTMLized"
      src="/static/work/xhtmlized.png"
    />
    <div
      className="experience-list__item-content"
    >
      <h5
        className="experience-list__item-date"
      >
        Oct 2007 — Aug 2011 · 4yrs 4mos
      </h5>
      <h3
        className="experience-list__item-title"
      >
        Front-end Web Developer
      </h3>
      <h4
        className="experience-list__item-company"
      >
        <span>
          at
        </span>
         XHTMLized
      </h4>
      <div
        className="experience-list__item-info"
      >
        <p>
          Working on converting client's design to html, wordpress or other templates.
        </p>
      </div>
    </div>
  </section>
  <section
    className="experience-list__item"
  >
    <img
      alt="Logo Agrisoft"
      src="/static/work/agrisoft.png"
    />
    <div
      className="experience-list__item-content"
    >
      <h5
        className="experience-list__item-date"
      >
        Jan 2005 — Jun 2007 · 2yrs 6mos
      </h5>
      <h3
        className="experience-list__item-title"
      >
        Web Developer
      </h3>
      <h4
        className="experience-list__item-company"
      >
        <span>
          at
        </span>
         PT. Agrisoft Citra Buana
      </h4>
      <div
        className="experience-list__item-info"
      >
        <p>
          Working on websites and desktop applications, usually related to map technology.
        </p>
      </div>
    </div>
  </section>
</div>
`;
